addons/website/i18n/nl.po

```diff
@@ -173,7 +173,7 @@ msgid ""
 "                        just one person.</b> If you try to write with a wide general\n"
 "                        audience in mind, your story will ring false and be bland.\n"
 "                        No one will be interested. Write for one person. If it’s genuine for the one, it’s genuine for the rest."
-msgstr "<b>Geweldige verhalen zijn voor iedereen, ook al zijn ze maar geschreven \nvoor één iemand. </p> Als u probeert te schrijven met een breed algemeen \npubliek in het achterhoofd, zal uw verhaal onwaar en saai klinken. \nHet zal niemand interesseren. Schrijf voor één persoon. Als het echt is voor de één, dan is het echt voor de rest."
+msgstr "<b>Geweldige verhalen zijn voor iedereen, ook al zijn ze maar geschreven \nvoor één iemand. </b> Als u probeert te schrijven met een breed algemeen \npubliek in het achterhoofd, zal uw verhaal onwaar en saai klinken. \nHet zal niemand interesseren. Schrijf voor één persoon. Als het echt is voor de één, dan is het echt voor de rest."
```

Comment:
don't do this here, your changes would be erased at the next weekly synchronisation, should be submitted on transifex